src/BackgroundProcess.php

```diff
@@ -59,14 +59,18 @@ public function __construct($command)
      * @param string $outputFile File to write the output of the process to; defaults to /dev/null
      *                           currently $outputFile has no effect when used in conjunction with a Windows server
      */
-    public function run($outputFile = '/dev/null')
+    public function run($command, $outputFile = '/dev/null')
```

Comment:
You did not update the documentation above the method about this new argument. Also it seems like a breaking change. Is this wanted?